### ISSUE TITLE
Plugin unable to detect streams on Afreecatv.com

### ISSUE THREAD

**Issue by RhysJB** [chrippa/livestreamer#1303]
The latest Afreecatv update on April 27th has caused the plugin to not be able to detect, any fix guys? Thanks heaps.

**Comment by tenminute** [comment]
Works fine for me. But the add-on I'm using with Firefox isn't. Was wondering WTH was going on this morning...

**Comment by RhysJB** [comment]
I'm on firefox atm and can't detect any stream on Afreecatv.com, so your suggestion is to use other browsers with the livestreamer add-on?

**Comment by RhysJB** [comment]
I keep getting the error even when trying to boot thru the cmd prompt: No plugin can handle URL: http://play.afreecatv.com/StreamerID

**Comment by BIZKIT551** [comment]
Yeah, livestreamer isn't opening the afreecatv.com streams anymore. 

> error: No plugin can handle URL: http://play.afreecatv.com/galsa

I've noticed in the past few weeks that afreecatv.com was down for maintenance a few times, which leads me to think that afreecatv.com have changed their addresses. 

I hope someone fixes this soon

**Comment by RhysJB** [comment]
They have sheduled weekly maintenance every Wednesday 5AM KST, the last one changed the address or their player probably..

**Comment by BIZKIT551** [comment]
@RhysJB wow that sucks. So I'm guessing we have to wait till someone finds that address and updates the plugin.

**Comment by BIZKIT551** [comment]
@RhysJB 
Instead of
`http://play.afreecatv.com/StreamerID`
try
`http://play.afreeca.com/StreamerID`
and it should work.

**Comment by tenminute** [comment]
@RhysJB This is the add-on for Firefox I'm talking about that's no longer working (not sure if you use this)...
https://addons.mozilla.org/en-US/firefox/addon/open-livestreamer/

But yes, like @BIZKIT551... it's probably due to the address change. I never actually noticed the change (or when it happened), but that must be it. That's probably why the add-on for Firefox isn't working. I copy and pasted some .txt files I have backed up and haven't had any issues with it.

Use the old address "afreeca.com" instead of with the "tv" as "afreecatv.com".
